lib/grocery_store.rb

```diff
@@ -0,0 +1,34 @@
+require_relative "billing"
+require_relative "grocery"
+class GroceryStore 
+  def initialize 
+    @grocery_orders = []
+  end
+
+  def take_order 
```

Comment:
Don't call method at the end of method in a chain. Call methods one after the other.